mlir/utils/jenkins/Jenkinsfile

```diff
@@ -91,15 +91,29 @@ void postProcessPerfRes() {
 pipeline {
     agent none
     parameters {
+        // Below should be set statically by Jenkins job
         booleanParam(name: 'nightly', defaultValue: params.nightly ?: false,
                      description: 'Run extra nightly-only tests')
         booleanParam(name: 'canXdlops', defaultValue: params.canXdlops ?: true,
                      description: 'Can this CI instance use xdlops (no for public server)')
+
+        // Each below control whether to run a individual stage from parallel run
+        // They default to true but deverloper can toggle them for debugging purpose
+        booleanParam(name: 'sharedLib', defaultValue: params.sharedLib ?: true,
+                     description: 'Run the shared library stage')
+        booleanParam(name: 'staticLib', defaultValue: params.staticLib ?: true,
```

Comment:
Unlike the static parameters above, the `params.staticlib ?:` setup shouldn't be used here - we really don't want these to be sticky

Reply:
Ha thanks for pointing out. Didn't know this. So to make sure I understand `params.sharedLib ?: true` will make the selection sticky? In terms that if I tried to manually set it to false, for example, then next time when CI job pick it up it will also be false?

Reply:
That's what I'm pretty sure that syntax does, yeah.

Reply:
Done in fce0368